go.mod

```diff
@@ -4,6 +4,7 @@ go 1.14
 
 require (
 	github.com/AndreasBriese/bbloom v0.0.0-20190823232136-616930265c33 // indirect
+	// agl/ed25519 only used in tests for backward compatibility, *do not* use in production code.
```

Comment:
I added this note to be clear that this dependency is only used in tests. 
Eventually, I guess we could stop caring about older implementations and directly remove the compatibility test thus this dependency. I think won't be a problem leaving this anyway since it would be weird that something use this dependency in production since is too crypto-focused.

Reply:
Will this ever get overwritten?

Reply:
I force-updated some dependency, and seems `go mod tidy` doesn't overwrite this, so will stay.